### DISCUSSION TITLE
ca-loader solved and viewer working

### DISCUSSION THREAD

**Discussion by gbengaoluwadahunsi** [camicroscope/caMicroscope#727] [Q&A] [answered]
Hello, everyone
@birm has sent the codes to update and fix the ca-loader
![slide](https://github.com/camicroscope/caMicroscope/assets/77133593/8c5c0b11-1aef-45da-88f2-d5b77846ad35)

I have tested it and it worked perfectly, here is it
1. To your already cloned and built Distro repo, run again
2. **docker-compose -f develop.yml build loader --no-cache**
3. Start again your app with **docker-compose -f caMicroscope.yml up**
4. navigate to the slides link and upload your image, it should not be a jpeg but svs
5.  After uploading, click open and you will be able to view the image in the image-viewer and play around with it.

Happy viewing
Enjoy the contributions

**Comment by ayush-0110** [comment]
Thanks for the update

**Reply by We13b-MD** [reply, marked as answer]
Thanks gbenga this really helped

**Reply by We13b-MD** [reply]
@gbengaoluwadahunsi where did you get your svs file

**Reply by gbengaoluwadahunsi** [reply]
here @We13b-MD  https://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/

**Reply by We13b-MD** [reply]
Ok thank you

**Reply by DIANAAJAMBO** [reply]
Thank you for this. @gbengaoluwadahunsi

**Reply by We13b-MD** [reply]
@gbengaoluwadahunsi after successfully uploading the slide image I closed my application and tried to work again but I could not get the ca-back to run gives an error of ca-back exited I was thinking of changing it to factory settings and build again this happened after I uploaded the image or if anybody is having this issues let me know.

**Comment by nwanduka** [comment]
I just tried this and I was able to upload a slide. Thanks for the update @gbengaoluwadahunsi.

**Comment by gbengaoluwadahunsi** [comment]
Awesome

Happy viewing

**Comment by Moliki-Salman** [comment]
Thanks @gbengaoluwadahunsi, its works

**Comment by gbengaoluwadahunsi** [comment]
Glad to hear it!

**Comment by gbengaoluwadahunsi** [comment]
Glad it worked

Let's go!

Happy contributions

**Comment by We13b-MD** [comment]
@gbengaoluwadahunsi 
![Screenshot 2024-03-13 145051](https://github.com/camicroscope/caMicroscope/assets/122805291/b963f2f5-c661-4241-8579-c55de8b0166c)
mine is working after updating thank you gbenga

**Comment by gbengaoluwadahunsi** [comment]
Glad to hear it!

**Comment by gbengaoluwadahunsi** [comment]
Hi, is your docker still working?

**Reply by We13b-MD** [reply]
Yes it is but after uploading the slide image if the ca-back container fails to run other containers run successfully it happened after uploading the sliding image

**Reply by birm** [reply]
What do you mean about the ca-back container; is something going wrong for you there? If so, please open an issue or new discussion!

**Comment by gbengaoluwadahunsi** [comment]
Can you rebuild the ca-load and start the app again?

**Reply by We13b-MD** [reply]
I did that but still gave the error ca-back exited am thinking of changing my docker to factory settings and build again so I can be able to make changes to the UI and integrate the dark mode functionality

**Comment by gbengaoluwadahunsi** [comment]
Yes, you can do that.

Please give us a feedback after doing above , if it persists, do send me a
log of the error message.

**Reply by We13b-MD** [reply]
Ok sure I will

**Comment by gbengaoluwadahunsi** [comment]
Noted.

**Reply by We13b-MD** [reply]
@gbengaoluwadahunsi still having issues was able to build again but now am after building am getting different containers in the docker desktop instead of the distro containers

**Comment by gbengaoluwadahunsi** [comment]
Try to delete the containers in docker, close your docker and reopen it.

**Reply by We13b-MD** [reply]
i deleted it already i want to reinstall docker again i should get it running this time i will let you know how it goes

**Comment by gbengaoluwadahunsi** [comment]
Ok, sounds good.

**Comment by We13b-MD** [comment]
Hello @gbengaoluwadahunsi @ayush-0110  i have been having issues pushing my code and anytime i push this is what i get:camicroscope:master and We13b-MD:TestfunctionsCaMicroscope are entirely different commit histories. please how do i reolve it

**Comment by We13b-MD** [comment]
@gbengaoluwadahunsi  i have been able so set it up again but having problem making changes locally how d i set the build to build local changes instead of hosting git versions in my develop.yml file am a bit confused
